Server/controllers/maintenanceWindowController.js

```
@@ -48,11 +53,38 @@ const createMaintenanceWindow = async (req, res, next) => {
+  }
+  try {
+    const maintenanceWindow = await req.db.getMaintenanceWindowById(
+      req.params.id
     );
+    return res.status(200).json({
+      success: true,
+      msg: successMessages.MAINTENANCE_WINDOW_GET_BY_ID,
+      data: maintenanceWindow,
+    });
+  } catch (error) {
+    error.service === undefined ? (error.service = SERVICE_NAME) : null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)